modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/am/integration/test/utils/base/APIMIntegrationConstants.java

```diff
@@ -49,4 +49,37 @@
     public static final String IS_API_NOT_EXISTS = "\"isApiExists\":false";
     public static final String AM_MONITORING_WEB_APP_NAME = "APIStatusMonitor";
 
+    public static final class APPLICATION_TIER {
```

Comment:
@imesh i used APPLICATION_TIER in uppercase since it is Constant to represent Application Tiers as a Group. Since it is a class do we need to use naming convention used for classes?